overhead_analysis/run_tests.sh

```diff
@@ -0,0 +1,83 @@
+# Run fio tests on different modes: 0 - blockhisto disabled; 1 - blockhisto enabled
+# Place the fio job files in directory "job_files"
+# Modify the varible BCC_script in this file to point to the bcc script that is going to run
+
+BCC_script=~/bcc/tools/blkrqhist.py
+
+if [ ! -d "job_files" -o -z "$(ls -A ./job_files/)" ] 
+then 
+    echo "No job files. Exit." 
+    exit 1 
+fi 
+if [ ! -d "single_jobs" ]
+then 
+    echo "Split job files ..."
+    python3 split_jobs.py
+fi 
+
+
+function do_test {
+    mkdir -p $outdir 
+    for jobfile in ./single_jobs/*      # jobfile=./single_jobs/xxx.fio
+    do 
+        echo "Run job file: $jobfile"
+        filename=${jobfile##*/}
+        sudo perf stat -a -o $outdir/${filename%fio}perf -- fio $jobfile --output=$outdir/${filename%fio}out --output-format="json" --idle-prof=system
+    done
+}
+
+
+if [ -z $1 ]
+then 
+    echo "Please input mode: 0 - blockhisto disabled; 1 - blockhisto enabled" 
+    exit 1
+fi 
+
+for cnt in 1 2
+do 
+    if [ $cnt -eq 1 -a $1 -eq 0 ]
+    then 
+        outdir=./output
+        echo "Run without tracing"
+
+    elif [ $cnt -eq 1 -a $1 -eq 1 ] 
+    then
+        outdir=./output-blkhist
+        echo "Run with block histogram enabled"
+
+    elif [ $cnt -eq 2 -a $1 -eq 0 ]
+    then
+        outdir=./output-ebpf
+        echo "Run with eBpf tracing"
+        sudo python $BCC_script &  # run ebpf script at backgroud 
+        pid=$!
+        sleep 3     # ensure tracing tool has started
```

Comment:
Is this enough time to ensure the program has been loaded?

Reply:
Since I tried to not modify the BCC script itself, I set it to 5 sec this time. Loading of the program can be verified by seeing the program's prompt appear before tests got run.